### DISCUSSION TITLE
What does this situation mean

### DISCUSSION THREAD

**Discussion by mortal-sec** [projectdiscovery/nuclei#4931] [Q&A]
.\nuclei.exe

[WRN] Found 18 template[s] loaded with deprecated paths, update before v3 for continued support.
[WRN] Found 1 templates with runtime error (use -validate flag for further examination)
[INF] Current nuclei version: v3.2.2 (latest)
[INF] Current nuclei-templates version: v9.7.8 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 126
[INF] Templates loaded for current scan: 7716
[INF] Executing 7733 signed templates from projectdiscovery/nuclei-templates
[INF] No results found. Better luck next time!

It's the WRN above

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
